brokerapi/brokers/models/service_broker.go

```diff
@@ -28,7 +30,7 @@ type ServiceBrokerHelper interface {
 	Bind(instanceID, bindingID string, details brokerapi.BindDetails) (ServiceBindingCredentials, error)
 	BuildInstanceCredentials(bindRecord ServiceBindingCredentials, instanceRecord ServiceInstanceDetails) (map[string]string, error)
 	Unbind(details ServiceBindingCredentials) error
-	Deprovision(instanceID string, details brokerapi.DeprovisionDetails) error
+	Deprovision(ctx context.Context, instance ServiceInstanceDetails, instanceID string, details brokerapi.DeprovisionDetails) error
```

Comment:
Is the instanceID param still necessary with the addition of the instance param?

Reply:
You are correct, it isn't necessary anymore.